docs/lab/overview/insurance.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Insurance Ordering"
+---
+
+Vital supports ordering with commercial insurance billing. To set this up, there are a set of steps required. This document explains the complete flow for ordering via commercial insurance.
+
+
+## Setting Insurance Data
+
+To place an order for a `user` in Vital with commercial insurance, you must first provide Vital with insurance data.
+
+This can be done via the [`POST /{user_id}/insurance`](/api-reference/user/create-insurance) endpoint, with the following payload:
```

Comment:
I would add that it is expected that insurance data will become outdated from time to time, and that we have the expectation that if the customer identifies a change in the insurance provider for a particular user, that it is updated on our servers to minimize risk of claims denials and patient balance billing by the lab partner.